automatic/flashplayeractivex/update.ps1

```diff
@@ -33,7 +33,7 @@ function global:au_GetLatest {
   $CurrentVersion = ( $try.Version )
   $majorVersion = ([version] $CurrentVersion).Major
 
-  $url32 = "https://fpdownload.macromedia.com/pub/flashplayer/pdc/${CurrentVersion}/install_flash_player_${majorVersion}_plugin.msi"
+  $url32 = "https://download.macromedia.com/pub/flashplayer/pdc/${CurrentVersion}/install_flash_player_${majorVersion}_plugin.msi"
```

Comment:
I know, I have tested it.
I was just so sure I had tried it, could be I had some kind of typo somewhere.